src/doc/DocHighlightThread.js

```diff
@@ -604,6 +604,37 @@ class DocHighlightThread extends AnnotationThread {
         }
         return this.pageEl;
     }
+
+    setThreadBoundary() {
+        if (!this.location || !this.location.quadPoints) {
+            return;
+        }
+
+        this.minX = Infinity;
+        this.minY = Infinity;
+        this.maxX = 0;
+        this.maxY = 0;
+
+        this.location.quadPoints.forEach((quadPoint) => {
+            const [x1, y1, x2, y2, x3, y3, x4, y4] = quadPoint;
+
+            if (Math.min(x1, x2, x3, x4) < this.minX) {
+                this.minX = Math.min(x1, x2, x3, x4);
+            }
+
+            if (Math.max(x1, x2, x3, x4) > this.maxX) {
+                this.maxX = Math.max(x1, x2, x3, x4);
+            }
+
+            if (Math.min(y1, y2, y3, y4) < this.minY) {
+                this.minY = Math.min(y1, y2, y3, y4);
+            }
+
+            if (Math.max(y1, y2, y3, y4) > this.maxY) {
+                this.maxY = Math.max(y1, y2, y3, y4);
+            }
+        });
+    }
```

Comment:
Wouldn't it be easier/cleaner to just do a min/max check, with assignment?

```
this.minX = Math.min(x1, x2, x3, x4, this.minX);
this.maxX = Math.max(x1, x2, x3, x4, this.maxX);
this.minY = Math.min(y1, y2, y3, y4, this.minY);
this.maxY = Math.min(y1, y2, y3, y4, this.maxY);
```
I understand the want to check beforehand, but it seems more complicated than it needs to be

Reply:
👍 this was an oversight on my part! Thanks!